### DISCUSSION TITLE
v0.15.1-dev

### DISCUSSION THREAD

**Discussion by sriramveeraghanta** [makeplane/plane#3633] [Announcements]
⚡ Experience lightning-speed views, now 20 times faster! Stay up-to-date on the go with our enhanced mobile-friendly version with optimized email notifications. Boost your productivity with improvements to keyboard accessibility and real-time updates to analytics.

## Improvements

- Your views now load faster; we've enhanced Kanban, list, and spreadsheet layouts, speeding up issue rendering. We'll soon extend this to other layouts. #3538

https://github.com/makeplane/plane/assets/9498163/42267835-217a-480c-9779-206f3dd9dde4

- Now ARM builds are available in docker hub. #3569
- The spreadsheet layout has been enhanced for increased productivity through the implementation of keyboard navigation, which improves accessibility. #3564

https://github.com/makeplane/plane/assets/9498163/377572e1-2c4c-48cd-b201-e6bfec4fef18

- Breadcrumbs were slow to navigate back; we have now fixed them to save your time. #3537
- Check your work at a glance anywhere you are with mobile-responsiveness extended to dashboard, analytics, modules and cycles. #3541, #3604, #3580
- Get notified via email as soon as you are assigned to a newly created issue. #3572
- Receive a reminder for unsaved changes in pages when you return to them and navigate away. #3577
- Dashboards will help you see priorities even if there are no due dates; just filter by "none" in them. #3556
- The burn-down chart is now consistent for cycles, even if issues are transferred to a new cycle. #3600
- Analytics are now real-time; change an issue and see your charts update instantly. #3559

## Bug Fixes

- Upgrading your plane instance had troubles, it's fixed now. #3571
- When you create an issue from dashboard, the project you select is the project issue get's created in consistently. #3549
- Sorting by priority and due dates in spreadsheet layout is fixed. #3550
- Edit start dates for modules which already had one. #3565
- Gantt layout now scrolls consistently. #3585
- Drag-and-drop in Kanban layouts now functions properly at the bottom of the page. #3550
- Fixed notification omission for first mentions. #3573

Thanks to all our contributors - @pablohashescobar, @gurusainath, @NarayanBavisetti, @AnmolBhatia1001, @rahulramesha, @prateekshourya29, @rameshkumarchandra, @1akhanBaheti, @aaryan610, @etaxi341 

<hr /><em>This discussion was created from the release <a href='https://github.com/makeplane/plane/releases/tag/v0.15.1-dev'>v0.15.1-dev</a>.</em>

**Comment by mmospanenko** [comment]
Thank you for ARM builds! Really important feature, we have all infrastructure on ARM, and it was blocking a little :)
